src/libs/ReportActionsUtils.ts

```diff
@@ -1348,6 +1350,34 @@ function isLinkedTransactionHeld(reportActionID: string, reportID: string): bool
     return TransactionUtils.isOnHoldByTransactionID(getLinkedTransactionID(reportActionID, reportID) ?? '-1');
 }
 
+function getAccountIDsFromMessage(message: string) {
+    const regex = /<mention-user\s+accountID="(\d+)"\/>/g;
+    const matches = [];
+    let match;
+    // eslint-disable-next-line no-cond-assign
+    while ((match = regex.exec(message)) !== null) {
+        matches.push(match[1]);
+    }
+    return matches;
+}
+
+function getMentionedEmailsFromMessage(message: string) {
+    const regex = /<mention-user>(.*?)<\/mention-user>/g;
+    const matches = [];
+    let match;
+    // eslint-disable-next-line no-cond-assign
+    while ((match = regex.exec(message)) !== null) {
+        matches.push(Str.removeSMSDomain(match[1].substring(1)));
+    }
+    return matches;
+}
+
+function didMessageMentionCurrentUser(message: string) {
```

Comment:
Let's have this take in an action directly, which will make it easier to use and be helpful for getting the mentionedAccountIDs.